docs/develop/01_sdk/02_cookbook/04_claiming/01_ctype_creation.md

```diff
@@ -8,35 +8,39 @@ import TsJsBlock from '@site/src/components/TsJsBlock';
 import CreateCType from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/claiming/01_create_ctype.ts';
 import FetchCType from '!!raw-loader!@site/code_examples/sdk_examples/src/core_features/claiming/02_fetch_ctype.ts';
 
+Every KILT credential has to conform to a CType.
+A CType describes which properties a credential has and what type these properties have.
+CTypes must be registered on the Spiritnet blockchain.
+To learn more about the CTypes, see the [CType concept section](../../../../concepts/05_credentials/02_ctypes.md)
```

Comment:
```suggestion
To learn more about CTypes, see the [CType concept section](../../../../concepts/05_credentials/02_ctypes.md)
```
I think you can use the absolute path here?

Reply:
[03c4bea](https://github.com/KILTprotocol/docs/pull/244/commits/03c4bea55ac4c369e9f612200aa4b4f713d5e3c8)